src/components/EarnDashboard/components/ProviderDetails/OpportunityRow.tsx

```diff
@@ -218,7 +219,7 @@ export const OpportunityRow: React.FC<
             icons={icons}
             subText={assetCellSubText}
             justifyContent='flex-start'
-            isExternal={opportunity.isReadOnly}
+            isExternal={opportunity.isReadOnly && !asset?.isPool}
```

Comment:
Fix to not show the read-only icon for read-only assets that route to swapper for Portals swapping